### PR TITLE
Add some website content blocks

### PR DESCRIPTION
**Pull request by dmlb2000**
This adds some content blocks to support the main website under construction.

Signed-off-by: David Brown <dmlb2000@gmail.com>

### CODE REVIEW

**Comment by dmlb2000** [comment]
@kariesue I've not seen any review on this yet, did you have time to look at this?

File: project/mission-vision.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+## Mission
+
+To provide institutions a foundation to support the scientific data
+life-cycle.
```
Comment (kariesue) [inline project/mission-vision.md]: lifecycle

File: project/mission-vision.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+## Vision
+
+We will become a focal point for institutions to collaborate, share
+scientific data, integrate new technologies and develop their
```
Comment (kariesue) [inline project/mission-vision.md]: comma after technologies

File: project/mission-vision.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+research portfolio. We will enable institutions to meet and exceed
+emerging sponsor data challenges, while maintaining the promises
```
Comment (kariesue) [inline project/mission-vision.md]: remove "the" before promises

File: project/mission-vision.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+kept in the past. We will provide the platform to accellerate
+scientific research in future directions never thought possible
```
Comment (kariesue) [inline project/mission-vision.md]: We will provide the platform to accelerate scientific research in directions never before thought possible.

File: project/mission-vision.md (hunk 1 above)
Comment (BenjaminStuermer) [reply project/mission-vision.md]: According to dictionaries I've checked on line, "life cycle" should be written as two words without a hyphen. (https://www.dictionary.com/browse/life-cycle)

File: project/website.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+completing and collaborative. Pacifica can accelerate the scientific
+process by making search and discovery tracta
+
+### How to Solve it
+
+Open Source does not stop at releasing the source code. Many problems
+require community to share ideas, processes and support features to
+solve the requirements. This is the natural evolution of Open Source
+to Open Everything.
+
```
Comment (kariesue) [inline project/website.md:30]: Open source does not end with the release of the source code. Complex problems require the community to share ideas, processes, and support features to solve the requirements -- the evolution from open source, to open everything.

Comment (kariesue) [reply project/website.md:30]: Needs a line about how Pacifica enables this community interaction ...

File: project/website.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+Pacifica supports the scientific data lifecycle.
+
+### Data Management
+
+Scientific Data Management systems for research and development
+institutions require many solutions to already complex problems.
+Pacifica has solutions to manage data between projects that are
+completing and collaborative. Pacifica can accelerate the scientific
+process by making search and discovery tracta
+
```
Comment (kariesue) [inline project/website.md:22]: Scientific data management systems for research and development institutions need to provide uncomplicated solutions to meet often complicated problems. Pacifica offers institutions a platform to collaboratively manage data between projects. 

... looks like unfinished section. LMK when to take another look at it.

File: project/website.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+Open Source does not stop at releasing the source code. Many problems
+require community to share ideas, processes and support features to
+solve the requirements. This is the natural evolution of Open Source
+to Open Everything.
+
+### Impact
+
+Pacifica seeks to enable institutions to reduce risk while
+accelerating scientific impact of the data they produce. Pacifica has
```
Comment (kariesue) [inline project/website.md]: is it scientific impact for everyone who might use Pacifica?

File: project/website.md [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+interfaces to allow scientists to automate workflow to ensure
```
Comment (kariesue) [inline project/website.md]: legal doesn't want us using the word "ensure." Consider safeguard, provide, ...

File: project/mission-vision.md (hunk 2 above)
Comment (dmlb2000) [reply project/mission-vision.md]: So are you advocating for an Oxford Comma? https://www.grammarly.com/blog/what-is-the-oxford-comma-and-why-do-people-care-so-much-about-it/

Comment (dmlb2000) [reply project/mission-vision.md]: But Weird Al told me not to (https://www.youtube.com/watch?v=8Gv0H-vPoDc)

Comment (kariesue) [reply project/mission-vision.md]: Oh yes! I loooooove the Oxford comma! :)

File: project/website.md (hunk 6 above)
Comment (dmlb2000) [reply project/website.md:22]: Yup I screwed up on this... didn't complete the section good catch.

File: project/mission-vision.md (hunk 1 above)
Comment (dmlb2000) [reply project/mission-vision.md]: Done.

File: project/mission-vision.md (hunk 2 above)
Comment (dmlb2000) [reply project/mission-vision.md]: Okay, good enough for me ;)

File: project/mission-vision.md (hunk 3 above)
Comment (dmlb2000) [reply project/mission-vision.md]: Done.

File: project/mission-vision.md (hunk 4 above)
Comment (dmlb2000) [reply project/mission-vision.md]: Done.

File: project/website.md (hunk 5 above)
Comment (dmlb2000) [reply project/website.md:30]: This should be done and ready for second review.

File: project/website.md (hunk 6 above)
Comment (dmlb2000) [reply project/website.md:22]: Done.

File: project/website.md (hunk 7 above)
Comment (dmlb2000) [reply project/website.md]: Done.

File: project/website.md (hunk 8 above)
Comment (dmlb2000) [reply project/website.md]: Done.

**Comment by dmlb2000** [comment]
@kariesue If you could review again, I should have fixed everything requested.

**Pull request merged**